addons/l10n_ar/models/account_chart_template.py

```diff
@@ -17,13 +17,26 @@ def _get_fp_vals(self, company, position):
         return res
 
     def _prepare_all_journals(self, acc_template_ref, company, journals_dict=None):
+<<<<<<< HEAD
         """ If Argentinean chart, we don't create sales journal as we need more
         data to create it properly """
+=======
+        """ If Argentinian chart, we modify the defaults values of sales journal to be a preprinted journal
+        """
+>>>>>>> ed015a3eec5... temp
```

Comment:
Hi @jco-odoo to solve this conflict the comment should be: "If Argentinean chart, we modify the defaults values of sales journal to be a preprinted journal"
I think with this change it will be solved the runbot error.